lib/greenbar/renderers/hipchat.ex

```diff
@@ -26,7 +26,12 @@ defmodule Greenbar.Renderers.HipChatRenderer do
     |> Enum.reduce([], &(render_attachment(&1, &2, attachment)))
     |> List.flatten
     |> Enum.join
-    rendered_body <> "<br/>"
+
+    if length(String.split(rendered_body, "<br/>")) - 1 > 1 do
```

Comment:
Did you consider using a regex to match `br` tags and count the number of matches?